src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -230,6 +230,7 @@ internal enum MessageID
         IDS_AsyncMethodBuilderOverride = MessageBase + 12805,
         IDS_FeatureImplicitImplementationOfNonPublicMemebers = MessageBase + 12806,
         IDS_FeatureLineSpanDirective = MessageBase + 12807,
+        IDS_FeatureFileScopedNamespace = MessageBase + 12808,
```

Comment:
May as well make this 12809. You're going to conflict with me no matter what, but at least the merge will be slightly easier.